Dalamud.Boot/dllmain.cpp

```diff
@@ -133,7 +133,9 @@ DWORD WINAPI InitializeImpl(LPVOID lpParam, HANDLE hMainThreadContinue) {
     // ============================== VEH ======================================== //
 
     logging::I("Initializing VEH...");
-    if (utils::is_running_on_wine()) {
+    if (g_startInfo.NoExceptionHandlers) {
+        logging::I("=> Exception handlers are disabled from DalamudStartInfo.");
```

Comment:
Please mark this as a warning - it's an exceptional case that we want extremely visible in logs.